onnxruntime/test/providers/cpu/nn/pool_op_test.cc

```diff
@@ -1264,6 +1264,25 @@ TEST(PoolTest, LpPool) {
   test.Run();
 }
 
+TEST(PoolTest, LpPoolWithNegativeNumbers) {
+  OpTester test("LpPool");
+
+  test.AddAttribute("p", static_cast<int64_t>(1));
+  test.AddAttribute("auto_pad", "");
+  test.AddAttribute("strides", std::vector<int64_t>{2});
+  test.AddAttribute("pads", vector<int64_t>{0, 0, 0});
```

Comment:
I think pads needs to have only 2 elements as the input is 1D. This is causing the test to break on Linux.

Reply:
done.